src/main/java/com/sda/zdjavapol107/kolkokrzyzyk/GameLogic.java

```diff
@@ -0,0 +1,127 @@
+package com.sda.zdjavapol107.kolkokrzyzyk;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GameLogic {
+
+    public static void start() {
```

Comment:
przepiszcie to tak zeby wszystko nie bylo tu statyczne, pamietajcie ze bedziecie chcieli w przyszlosci przekazac do gdy sessionFactory zeby korzystac z bady danych i ewentualnie wraper na io zeby pisac testy :D